metadata/overlays/openshift/metadata-db-deployment.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: db
+  name: metadata-db
```

Comment:
Looks a lot like the same change in PR #25

Reply:
@crobby yes right, we can merge this PR since it has two changes and delete yours :-)